src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle {
+    double kat1;

```

Comment:
Avoid using non-descriptive field names like 'kat1'. Use names that clearly describe the purpose of the field, such as 'leg1'.